src/app/shared/components/template/components/text/text.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, OnInit } from "@angular/core";
+import { TemplateBaseComponent } from "../base";
+import { getStringParamFromTemplateRow } from "../../../../utils";
+
+interface ITextParams {
+  style: string | null;
+  textAlign: string | null;
+  type: string;
+}
+
+@Component({
+  selector: "plh-tmpl-text",
+  templateUrl: "./text.component.html",
+  styleUrls: ["../tmpl-components-common.scss"],
+})
+export class TmplTextComponent extends TemplateBaseComponent implements OnInit {
+  params: Partial<ITextParams> = {};
+  isFalsy: boolean;
```

Comment:
nit(non-blocking): I can appreciate this accurately describes the value, but not so much what it actually impacts. Maybe something more like `hasTextValue` would be clearer.